### DISCUSSION TITLE
Kibana log is constantly saying license information could not be obtained

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#7863] [Unsupported Versions]
Kibana log is showing this on me.  Running most recently 120 version.

","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:15:07.278+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:15:11.342+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:15:37.279+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:15:41.341+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:16:07.279+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:16:11.341+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:16:37.280+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}
{"ecs":{"version":"1.12.0"},"@timestamp":"2022-04-26T17:16:41.344+00:00","message":"License information could not be obtained from Elasticsearch due to ConnectionError: connect ECONNREFUSED 192.168.20.57:9200 error","log":{"level":"WARN","logger":"plugins.licensing"},"process":{"pid":294}}

**Comment by dougburks** [comment]
Is Elasticsearch running?

Have you checked the Elasticsearch log for additional clues?

**Reply by 3isenHeiM** [reply]
I have the exact same error, and nothing peculiar is present in the ElasticSearch logs (there are no timestamps in the ElasticSearch logs :/ )

The only lines I have are :

```
ERROR: Elasticsearch exited unexpectedly
Importing PKCS12 keypair into Java keystore
Importing keystore /usr/share/elasticsearch/config/elasticsearch.p12 to /usr/share/elasticsearch/config/sokeys...
Warning: Overwriting existing alias 1 in destination keystore
Entry for alias 1 successfully imported.
Import command completed:  1 entries successfully imported, 0 entries failed or cancelled

```
